sourcecode/apis/contentauthor/tests/Unit/Models/H5PContentTest.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace Tests\Unit\Models;
+
+use App\H5PContent;
+use App\H5PContentsMetadata;
+use App\H5PLibrary;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Http\Request;
+use Tests\TestCase;
+use Tests\Traits\WithFaker;
+
+class H5PContentTest extends TestCase
+{
+    use RefreshDatabase;
+    use WithFaker;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+        $this->refreshDatabase();
```

Comment:
This method is already called via the base test class (see `Illuminate\Foundation\Testing\TestCase::setUpTraits`). The effect here would be that the database is refreshed twice.